config/initializers/seeds.rb

```diff
@@ -0,0 +1,51 @@
+return unless (DatabaseSeed.table_exists? rescue(false)) && !Foreman.in_rake?('db:rollback')
+
+Foreman::Application.configure do |app|
+  config.after_initialize do
+    foreman_seeds = Dir.glob(Rails.root + 'db/seeds.d/*.rb')
```

Comment:
Does root have a trailing slash? Would `File.join` be better? Same below for the glob

Reply:
Possibly. This works today as this code is largely borrowed from `db/seeds.rb` to test the implementation.